addon/mixins/route.js

```diff
@@ -245,8 +262,8 @@ export default Ember.Mixin.create({
    @return {Object}
    */
   _includeBoundParams: function(options, boundParams) {
-    if (Ember.keys(boundParams).length > 0) {
-      Ember.keys(boundParams).forEach( (key) => {
+    if (keys(boundParams).length > 0) {
```

Comment:
I would stick to using 
```js
Ember.isEmpty()
```
as you have done here https://github.com/hhff/ember-infinity/pull/48/files#diff-95eb5c8003a4fd239be76137499e5176R146